lib/watir/elements/select.rb

```diff
@@ -106,18 +106,6 @@ def selected?(str_or_rx)
       raise(UnknownObjectException, "Unable to locate option matching #{str_or_rx.inspect}")
     end
 
-    #
-    # Returns the value of the first selected option in the select list.
-    # Returns nil if no option is selected.
-    #
-    # @return [String, nil]
-    #
-
-    def value
-      option = selected_options.first
-      option&.value
-    end
```

Comment:
Are you fine that this removal would technically be a change in behaviour for multi-selects? A multi-select with nothing selected would return `nil` in 6.17 but will now return `""` (empty string) for 6.18.

The change seems reasonable in the long term. It's just whether we should go through the whole deprecation warning for this.

Reply:
erf, no, I don't want to do this, actually. I eventually want to implement this with `WebDriver::Element#property` which will return null instead of empty string. I don't want to go down that path, though until we also get `WebDriver::Element#dom_attribute` which won't be until Selenium 4 beta 2, so I figured I'd wait on all of it until Watir 7. I'm pulling the last commit and keeping this method. I added a spec for this as well.